### ISSUE TITLE
Documentation unclear about features

### ISSUE THREAD

**Issue by michaelWuensch** [alexzhirkevich/custom-qr-generator#33]
**Describe the option you are trying to implement**
Hi, I try to create a QR code with random circle sizes and colors with the vector implementation.
Is this possible? The instagram example is only provided with the deprecated method.

It feels like this should be simple, but I can't find a way how to do this.
Thanks for your help.

P.S: If it is not possible yet I would love to see this feature implemented.

**Comment by alexzhirkevich** [comment]
You can do it the same way as for deprecated one:

```kotlin
class RandomPixelShape(
    private val from : List<QrVectorPixelShape>
) : QrVectorPixelShape {
    override fun createPath(size: Float, neighbors: Neighbors): Path {
        return from.random().createPath(size, neighbors)
    }
}
```

```kotlin
 shapes {
    darkPixel = RandomPixelShape(
        listOf(
            QrVectorPixelShape.Circle(.3f),
            QrVectorPixelShape.Circle(.5f),
            QrVectorPixelShape.Circle(.7f),
        )
    )
}
```

<img width="303" alt="image" src="https://github.com/alexzhirkevich/custom-qr-generator/assets/63979218/3db41533-776d-49ef-a0c0-a0c79e85b7e0">

**Comment by michaelWuensch** [comment]
Thanks, this worked! I now wanted to try to make random solid colors (3 slightly different variations).
I tried to make it similar like your example above, but for some reason always one color is chosen randomly for the whole QR code instead of the pixels.
Does CreatePaint not get executed for every pixel?

**Comment by alexzhirkevich** [comment]
>Random colors for individual pixels are not available at the moment.

Well...

>Does CreatePaint not get executed for every pixel?

All pixels are combined to a single path before applying a paint so you can make them gradient

**Comment by michaelWuensch** [comment]
Ok, thanks, so it is not possible with the vector method. Damn.
Thanks for your help anyway.

**Comment by alexzhirkevich** [comment]
Will be possible after #35. 

You can test it now in a `com.github.alexzhirkevich:custom-qr-generator:separate_painting-SNAPSHOT` snapshot and give a feedback

**Comment by michaelWuensch** [comment]
Thanks, that one was fast!
I am looking forward to the next release!

**Comment by alexzhirkevich** [comment]
Available in `2.0.0-alpha01`
